presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -326,7 +326,30 @@ public static PlanMatchPattern join(JoinNode.Type joinType, List<ExpectedValuePr
                         joinType,
                         expectedEquiCriteria,
                         expectedFilter.map(predicate -> rewriteIdentifiersToSymbolReferences(new SqlParser().createExpression(predicate))),
-                        expectedDistributionType));
+                        expectedDistributionType,
+                        Optional.empty()));
+    }
+
+    public static PlanMatchPattern join(
```

Comment:
It seems to be unused in this commit, therefore it should go to the commit that uses it (ApplyDynamicFilters)